### PR TITLE
[TASK] Convert icons to SVG

### PR DESCRIPTION
**Pull request by Braunson**
As per #1522, 

- Converted gif/png icons to SVG format
- Updated CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Add an `AccommodationOption` model (#1731, #1736)
 
 ### Changed
+- Converted icons from gif/png to SVG (#1522)
```
Comment (oliverklee) [inline CHANGELOG.md:15]: This should be the ID of the PR, not of the ticket. (This allows people to click on the ID in the changelog and directly see the changes.)

Also, please move this to the `Added` section.

**Review by oliverklee — CHANGES_REQUESTED** [review]
Thank you for this contribution! :heart: :cookie:

You'll also need to update all references to these files. You can search for `.png` and `.gif` in these files/folders:
- `Configuration/TCA/`
- `Classes/BackEnd/EventsList.php`
- `Resources/Public/CSS/BackEnd/BackEnd.css`
- `ext_localconf.php`
- `ext_tables.php`

**Comment by Braunson** [comment]
@oliverklee Ah yes, the references have been updated, so has the CHANGELOG.

**Comment by oliverklee** [comment]
The SVG icons look *very* strange in my backend (see screenshot) - they seem to have only one (foreground) color and seem to be very smooth so that most of the details are lost.
grafik

How did you create/convert the icons?

**Review by oliverklee — CHANGES_REQUESTED** [review]
I think something has gone wrong during the conversion to SVG.

**Comment by oliverklee** [comment]
If you like, we can also do this in smaller PRs (for which you'll need to use git branches for the PRs) so we can get this merged more quickly:
- BE module icon
- content wizard icon
- button icons in the BE module (canceled, confirmed, edit, hide, new, print, unhide)

**Pull request closed without merging**